torch/onnx/__init__.py

```diff
@@ -30,7 +30,7 @@ def _export(*args, **kwargs):
 def export(model, args, f, export_params=True, verbose=False, training=False,
            input_names=None, output_names=None, aten=False, export_raw_ir=False,
            operator_export_type=None, opset_version=None, _retain_param_name=True,
-           do_constant_folding=False, example_outputs=None, strip_doc_string=True,
```

Comment:
Can you change it to None.

And we enable it if it's not ONNX_ATEN_FALLBACK? otherwise use False. Internal tests are broken. I think some of them relies on the input numbers.

Reply:
Why do we want a None default?
I can set it to False for ATEN_FALLBACK.

Reply:
Yes, let's do that.